tsfresh/utilities/string_manipulation.py

```diff
@@ -1,6 +1,13 @@
+# -*- coding: utf-8 -*-
+# This file as well as the whole tsfresh package are licenced under the MIT licence (see the LICENCE.txt)
+# Maximilian Christ (maximilianchrist.com), Blue Yonder Gmbh, 2016
+
 import ast
 import numpy as np
 from six import string_types
+import ipaddress
```

Comment:
this is not needed anymore. Please remove :-)

Reply:
Eagle eyes, Done!